concepts/comparisons/about.md

```diff
@@ -1,2 +1,184 @@
+
+Also unlike `C`, expressions like `a < b < c` have the interpretation that is conventional in mathematics.
+
+```python
+>>> x = 2
+>>> y = 5
+>>> z = 10
+>>> x < y < z
+True
+>>> x < y > z
+False
+>>> x > y < z
+False
+```
+
+## Comparison between different data types
+
+Since everything in Python is an `object`, things can get interesting when objects of different types are compared. For example, the `str` value of a number is considered completely different from the `integer` or `floating-point` value. However, an `integer` **can** be considered equal to a `float`, as they are both numeric types that Python can implicitly convert to compare. For other numeric types, comparison operators are defined where they "make sense", but throw a `TypeError` if the underlying objects cannot be converted for comparison. For more information on the rules that python uses for numeric conversion, see [arithmetic conversions][arithmetic conversions] in the Python documentation.
+
+```python
+>>> 17 == '17'
+False
+>>> 17 == 17.0
+True
+>>> 17.0 == 0017.000
+True
+>>> complex(1, 2) == complex(1.0, 2.00)
+True
+```
+
+Python makes this distinction because strings are text, while `integers` and `floats` are numeric types.
+
+## Value Comparison
+
+The operators `<`, `>`, `==`, `>=`, `<=`, and `!=` compare the _values_ of two different objects. The objects do not need to have the same type. Remember that in Python
+
+every object has a `value` in addition to `type` and `identity`.
+
+### Following are comparison behaviour of most `built-ins` types.
+
+Numbers of built-in _numeric_ types such as `int`, `hex`, `ocal`, `binary`, `float`, `complex` and of the standard library types `fractions.Fraction` and `decimal.Decimal` can be compared within and across their types.
+
+Any ordered comparison of a number to a `NaN` (_not a number_) value is `False`. A counter-intuitive implication is that `NaN` never compares equal to `NaN`.
+
+```python
+>>> x = float('NaN')
+>>> x
+nan
+>>> 3 < x
+False
+>>> x < 3
+False
+>>> x == x
+False
+```
+
+Strings (`str`) are compared _lexicographically_ using their individual numerical Unicode code points (_the result of passing each code point in the `str` to the built-in function `ord()`_). `str` and `binary` sequences cannot be directly compared.
+
+```python
+>>> 'santa' < 'claus'
+False
+>>> 'Santa' < 'claus'
+True
+>>> ord('s')
+115
+>>> ord('S')
+83
+>>> ord('c')
+99
+>>> '龙波' < '王想'  # chinese words

```

Comment:
@gurupratap-matharu  -- Please put the translation of the words in the comment.  Please also capitalize the language -- is this Mandarin or Cantonese or another Sinitic language?

Reply:
@BethanyG  -- Sure! Earlier these words were abstract names.


- Now I've changed them to be common words
- Their English translations has been added
- You can verify these words on google translate. Just put language detection to be automatic.

Reply:
@gurupratap-matharu  - I am not seeing a change to the text here?  Did you push a change?